bool left_encoder_ok    # left encoder status 
bool right_encoder_ok   # right encoder status 
